src/main/java/duke/internal/Parser.java

```diff
@@ -0,0 +1,145 @@
+package duke.internal;
+
+import java.util.Arrays;
+import java.util.stream.IntStream;
+
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.DeadlineCommand;
```

Comment:
maybe you shouldn't leave a space here for consistency purposes?

Reply:
Hi! Just wanted to point out that the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements) actually uses newlines to separate packages.

The AB3 checkstyle also enforces this newline:
 
<img width="871" alt="Screenshot 2022-09-03 at 13 13 05" src="https://user-images.githubusercontent.com/50949210/188256843-951805a2-2f06-4da4-9f08-089f4430032d.png">

Reply:
ah i see! my bad